### DISCUSSION TITLE
Newlines as line breaks and external links with "target="_blank"

### DISCUSSION THREAD

**Discussion by OrangeBurrito** [pngwn/MDsveX#279] [Q&A] [answered]
Could there be an option to render newlines as </br/> tags and set external links to target="_blank" like [markdown-it](https://markdown-it.github.io/) does?

**Comment by pngwn** [comment, marked as answer]
Setting external links to `target="_blank"` should be handled by [a plugin](https://github.com/remarkjs/remark-external-links).

Rendering newlines as a `<br>` can also be handled by [a plugin](https://github.com/remarkjs/remark-breaks).

You can read more about using plugins in [the docs](https://mdsvex.com/docs#remarkplugins--rehypeplugins).
